whois-commons/src/main/resources/patch/internals-1.111.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE `outgoing_message` (
      `message_id` varchar(80) NOT NULL,
      `email` varchar(80) NOT NULL,
      `last_update` datetime DEFAULT now(),
-     PRIMARY KEY (`message_id`)
+     PRIMARY KEY (`message_id`, `email`)
```

Comment:
Why this change requires email column?

Reply:
1 mail can have multiple recipients. So we can have two records with same message_id

Reply:
Sorry I was not expecting so many changes for this? Seems like a seperate PR which says support multiple recipients as it needs changes in Dao layer and then a seperate PR for this?